### ISSUE TITLE
Progress Bar values not announced on Screen Reader

### ISSUE THREAD

**Issue by grgr-dkrk** [react-native-progress-view/progress-bar-android#61]
from
https://github.com/facebook/react-native/issues/30845

> Androidのプログレスバーのアクセシビリティは、AccessibilityNodeInfoのRangeInfoクラスに大きく依存しています。このクラスは、最小値、最大値、および現在の値を定義し、プログレスバーコンポーネントのAccessibilityNodeInfoにアタッチする必要があります。

> 現在、これはaccessibilityValueプロパティを使用している場合はすべて機能しますが、組み込みクラスはこれを自動的に実行しません。

**Comment by amarlette** [comment]
Thank you for opening this up! There is also the issue of facebook/react-native#30941 which is in a similar vein of the values not announcing, if the progress bar is disabled then it doesn't announce. I've created a seperate issue #62

facebook/react-native#30840 is the parent issue for facebook/react-native#30941 and there are more specific details there

Any help in identifying if this issue has been addressed in this repo would be greatly appreciated!

**Comment by grgr-dkrk** [comment]
@amarlette
This component itself is a BaseView extends and works fine if there are props such as `accessibilityValue` on the JavaScript side, but the built-in side doesn't seem to support it.
If necessary, I can work on this issue, after getting permission from the maintainer. Thank you.

**Comment by grgr-dkrk** [comment]
I attached Delegate to this component and I was able to see Talkback read aloud.
I will report some issues.

--There are no minimum or maximum props for this component. react-native-web treats `0-100` as a range.
--Android 7 and lower operating systems that do not support RangeInfo did not use TalkBack to read this percentage.
--If `indeterminate` is true and` accessibilityValue` is set, it will appear to loop through the percentages from min to max. I think It needs props to pass `accessibilityValue.text` param instead.
